core/components/atoms/metricInput/MetricInput.tsx

```diff
@@ -177,6 +182,11 @@ export const MetricInput = React.forwardRef<HTMLInputElement, MetricInputProps>(
     })
   );
 
+  const aroowIconsClass = classNames({
+    ['MetricInput-arrowIcons']:true,
+    ['d-none']: !withArrowIcons,
```

Comment:
Add `MetricInput-arrowIcons--visible` class and add `display: none` in css file.